src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/breadcrumbData/breadcrumbDataException.tsx

```diff
@@ -30,11 +36,61 @@ const BreadcrumbDataException = ({breadcrumb}: Props) => {
         })}
       {breadcrumb?.message &&
         getBreadcrumbCustomRendererValue({
-          value: breadcrumb.message,
+          value: (
+            <FormatMessage
+              event={event}
+              orgId={orgId}
+              breadcrumb={breadcrumb}
+              message={breadcrumb.message}
+            />
+          ),
           meta: getMeta(breadcrumb, 'message'),
         })}
     </BreadcrumbDataSummary>
   );
 };
 
+function isEventId(maybeEventId: string): boolean {
+  // maybeEventId is an event id if it's a hex string of 32 characters long
+  return /^[a-fA-F0-9]{32}$/.test(maybeEventId);
+}
+
+const FormatMessage = withProjects(function FormatMessageInner({
+  event,
+  message,
+  breadcrumb,
+  projects,
+  loadingProjects,
+  orgId,
+}: {
+  event: Event;
+  projects: Project[];
+  loadingProjects: boolean;
+  breadcrumb: BreadcrumbTypeDefault;
+  message: string;
+  orgId: string | null;
+}) {
+  if (
```

Comment:
I would prefer an early return over full-body conditional wrapping... but it's just a suggestion 😉